spec/features/admin/admin_can_change_its_user_properties_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.